src/debugger/mediator.ts

```diff
@@ -29,7 +29,13 @@ export class Mediator {
 
 				let lines: string[] = parameters;
 				lines.forEach((line) => {
-					this.output.appendLine(line);
+					let message_content: string = line[0];
+					//let message_kind: number = line[1];
```

Comment:
This PR fixed the issue for me, but why do you left this commented code?

Reply:
In case futur contributor wonders why `message_content` is `line[0]`.

Reply:
I understand text comments, but `//let message_kind: number = line[1];` look weird :)

Reply:
No i was indeed talking about that comment line. If i dont comment it out there is a warning saying "unused variable".

Reply:
Got it, thanks :)